### ISSUE TITLE
examples from within a generator module must be run from the root

### ISSUE THREAD

**Issue by verdverm** [hofstadter-io/hof#103]
I should be able to run `hof gen` from a nested example directory and have it work

Problem synopsis

```
example/             // imports gen 

gen/...                 // points into dirs like below, relative to the module root

templates/...
partials/...
static/...
```

When hof is run from the module root, this works fine. When hof is run from a subdir, those relative paths should include a `../` for each directory we are nested from the root.

Not sure if it is better to use relative or just rework what the prefix should be...

```
$ '
> missing out in "loading shadow"
> 
> need to account for subdir from 
> 
> should we put .hof next to cue.mod?
> 
> what if there is none...
> 
> also check / fix when no cue.mod found in current changes
> 
> good first hire issue, add in fs.FS to simplify things
> 
> this subdir issue effects template loading and shadow resolution (if we consolidate .hof when cue.mod found)
```
---

### Issue with shadow and subdir

```
tony at penguin in ~/hof/mods/openapi/test/dev on main*
$ hof gen -v2
CueDirs: /home/tony/hof/mods/openapi /home/tony/hof/mods/openapi/test/dev ../..
Loading CUE from: []
Loading Generator: go

                You are running the 'go' generator
                        with PackageName: ""

                Note, that when running hof from inside a generator module,
                it currently must be run from the root.

                See GitHub issue: https://github.com/hofstadter-io/hof/issues/103
                Creating Adhoc Generator
runGen.doGen: fast: true
genOnce.doGen: fast: true
Loading Generator: go

                You are running the 'go' generator
                        with PackageName: ""

                Note, that when running hof from inside a generator module,
                it currently must be run from the root.

                See GitHub issue: https://github.com/hofstadter-io/hof/issues/103
                Creating Adhoc Generator
Loading shadow @ ".hof/shadow/go"
  adding: out/Error.go
  adding: out/NewPet.go
  adding: out/Pet.go
  adding: out/main.go
Writing output
tony at penguin in ~/hof/mods/openapi/test/dev on main*

```

**Comment by verdverm** [comment]
I think this impacts the shadow dir...

**Comment by verdverm** [comment]
Fixed in 0.6.5
